### PR TITLE
NO-ISSUE: Parameterize equinix secrets and change their names

### PR DESCRIPTION
**Pull request by danmanor**
Parameterize `equinix` secrets and change their names to fit the convention

### CODE REVIEW

**Comment by danmanor** [comment]
@rccrdpccl

File: openshift/template.yaml [hunk 1]
```diff
@@ -62,13 +62,13 @@ objects:
               - name: EQUINIX_PROJECT_ID
                 valueFrom:
                   secretKeyRef:
-                    key: equinix_project_id_key
-                    name: equinix_project_id
+                    key: ${PROW_JOBS_SCRAPER_EQUINIX_PROJECT_ID_KEY}
+                    name: ${PROW_JOBS_SCRAPER_EQUINIX_PROJECT_ID}
```
Comment (rccrdpccl) [inline openshift/template.yaml:66]: Maybe we could've had only one secret something like
`$PROW_JOBS_SCRAPER_EQUINIX_SECRET_NAME`
and keys `$PROW_JOBS_SCRAPER_EQUINIX_PROJECT_ID_KEY` and `$PROW_JOBS_SCRAPER_EQUINIX_TOKEN_KEY`.
This way we just have one secret to maintain, as the values seem related.

WDYT? Are we still in time for this change?

Comment (danmanor) [reply openshift/template.yaml:66]: We can do it, but all the other project's secrets are one key per secret, shouldn't we follow the others?

Comment (danmanor) [reply openshift/template.yaml:66]: What do you think is better?

Comment (rccrdpccl) [reply openshift/template.yaml:66]: We have some examples of secrets that have multiple keys, off the top of my head:
* kafka secrets (contain client_id, client_secret, url, metrics url, etc)
* opensearch secret (contain admin user and pwd, url)
* aws generated secrets in general

In my opinion it's easier to manage one secret, but if there are strong enough reasons to use 1 secret one key, I see no issues with that.

Comment (danmanor) [reply openshift/template.yaml:66]: Ack. I will change it

**Review by adriengentil — APPROVED** [review]

**Pull request merged**